Libraries/Connect/Public/Implementation/Clients/URLSessionHTTPClient.swift

```diff
@@ -195,15 +194,15 @@ extension Code {
     static func fromURLSessionCode(_ code: Int) -> Self {
         // https://developer.apple.com/documentation/cfnetwork/cfnetworkerrors?language=swift
         switch Int32(code) {
-        case CFNetworkErrors.cfurlErrorUnknown.rawValue:
+        case -998:
```

Comment:
I think it would be nice to retain at least at comment that references the constant we are using here. Like this:

```suggestion
        // CFNetworkErrors.cfurlErrorUnknown.rawValue
        case -998:
```